src/swarm/neo/node/ConnectionHandler.d

```diff
@@ -83,6 +83,7 @@ class ConnectionHandler : IConnectionHandler
 
         ***********************************************************************/
 
+        deprecated("Use new request handling framwork. (This symbol should have been removed in v5.0.0.)")
```

Comment:
Typo `framwork`

Reply:
Updated. (Though framworks are definitely the future.)